Source/Scene/ModelOutlineLoader.js

```diff
@@ -307,6 +307,7 @@ function addOutline(
         // Hackily update it, or else we'll end up creating the wrong type
         // of index buffer later.
         loadResources.indexBuffersToCreate._array.forEach(function (toCreate) {
+          if (toCreate === undefined) return;
```

Comment:
We use the function `Souce/Core/defined` to check if a variable is defined, as it also checks for `null`. So according to the [Coding Guide](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodingGuide), this is better written as:

```suggestion
          if (!defined(toCreate)) { 
              return;
          }
```